index.html

```diff
@@ -12,19 +12,34 @@
 
 		<!-- Theme used for syntax highlighted code -->
 		<link rel="stylesheet" href="plugin/highlight/monokai.css">
+		<style>
+			.front-page {
+				background-image: url("templates/assets/background.avif");
+			}
+			.front-page h1 {
+				color: #000000;
+			}
+			.second-page {
+				background-color: #6f4545;
+				background-size: cover;
+				background-position: center;
+				background-repeat: no-repeat;
+			}
+		</style>
```

Comment:
can those things be part of the template file?

Reply:
Yeah. That will go in the template file.